configs/BROTHERHOBBYF435/config.h

```diff
@@ -0,0 +1,138 @@
+
+#define FC_TARGET_MCU     AT32F435G
+
+#define BOARD_NAME        BROTHERHOBBYF435
+#define MANUFACTURER_ID   BROT
+
+// LEDS
+#define BEEPER_PIN             			 	PC15
+#define BEEPER_INVERTED				
+#define LED_STRIP_PIN          			 	PC9
+#define LED0_PIN               			 	PC13
+#define LED0_INVERTED
+
+// MOTORS & SERVOS
+#define MOTOR1_PIN              			PA8
+#define MOTOR2_PIN              			PA9
+#define MOTOR3_PIN              			PB10
+#define MOTOR4_PIN              			PB11
+#define MOTOR5_PIN              			PB7
+#define MOTOR6_PIN              			PB6
+#define MOTOR7_PIN              			PB0
+#define MOTOR8_PIN              			PB1
+
+// SPI			
+#define SPI1_SCK_PIN            			PA5
+#define SPI1_SDI_PIN            			PA6
+#define SPI1_SDO_PIN            			PA7
+
+#define SPI2_SCK_PIN            			PB13
+#define SPI2_SDI_PIN            			PB14
+#define SPI2_SDO_PIN            			PB15
+
+#define SPI3_SCK_PIN            			PB3
+#define SPI3_SDI_PIN            			PB4
+#define SPI3_SDO_PIN            			PB5
+
+// GYRO & ACC			
+#define USE_ACC
+#define USE_GYRO
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACCGYRO_BMI270
+#define ENSURE_MPU_DATA_READY_IS_LOW
+#define GYRO_1_EXTI_PIN         			PC4
+#define GYRO_1_CS_PIN           			PA4
+#define GYRO_1_SPI_INSTANCE                 SPI1
+#define GYRO_1_ALIGN                        CW90_DEG
+
+// OSD
+#define USE_MAX7456
+#define MAX7456_SPI_CS_PIN      			PB12
+#define MAX7456_SPI_INSTANCE                SPI2   
+
+// FLASH
+#define USE_FLASH
+#define USE_FLASH_W25N
+#define USE_FLASH_W25Q128FV
+#define USE_FLASH_M25P16
+#define FLASH_CS_PIN            			PA15
+#define FLASH_SPI_INSTANCE                  SPI3
+#define DEFAULT_BLACKBOX_DEVICE             BLACKBOX_DEVICE_FLASH
+
+// BARO & MAG
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define I2C1_SCL_PIN                        PB8
+#define I2C1_SDA_PIN                        PB9
+#define BARO_I2C_INSTANCE                   (I2CDEV_1)
+#define MAG_I2C_INSTANCE                    (I2CDEV_1)
+
+// UART & VCP
+#define UART1_RX_PIN            			PA10			
+#define UART2_TX_PIN            			PA2
+#define UART2_RX_PIN            			PA3
+#define UART3_TX_PIN            			PC10
+#define UART3_RX_PIN            			PC11
+#define UART4_TX_PIN            			PA0
+#define UART4_RX_PIN            			PA1
+#define UART5_TX_PIN            			PC12
+#define UART5_RX_PIN            			PD2
+#define UART6_TX_PIN            			PC6
+#define UART6_RX_PIN            			PC7
+
+// ADC			
+#define ADC_VBAT_PIN            			PC0
+#define ADC_RSSI_PIN            			PC5
+#define ADC_CURR_PIN            			PC1
+#define ADC_INSTANCE        				ADC1
+#define ADC1_DMA_OPT        				11
+#define DEFAULT_CURRENT_METER_SOURCE        CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE        VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE         386
```

Comment:
Is this an AIO, or does it include an ESC?

Reply:
> Is this an AIO, or does it include an ESC?

ESC，not AIO.